README.md

```diff
@@ -0,0 +1,83 @@
+# Barefoot Nomad(the Spinners)
+
+[![Coverage Status](https://coveralls.io/repos/github/andela/the_spinners-backend/badge.svg?branch=develop)](https://coveralls.io/github/andela/the_spinners-backend?branch=develop)
+
+ 
+ > A platform to make company global travel and accommodation easy and convenient for the 
+ strong workforce of savvy members of staff, by leveraging the modern web.
+
+ ## Usage example
+
+>This App will be the center piece for travelling. 
+Travellers will be able to register online and will see accomodation listing, recommended visits place.
+travellers will be able to chat in the app
+Selected sellers will be able to list their accomodation and other facilitiesl
+Travellers will be able to comment
+
+
+## Technologies used
+
+The project use:  
+### Backend
+ `Node Express` : the core back end technology  
+ `Heroku`: Used to host our app online
+ `Es6+` Following ES6 syntax. and AirBnB styling guide  
+ `PostgreSQL` Is used as our database management tool  
+ `Swagger` Used for API documentation.  
+ `Pivotal Tracker` A project management tool used to manage the app.  
+ `Npm` Used as the package manager for the app. A fast, reliable, and secure dependency management system.  
+
+### Front end
+ `Figma` : Use to protype the User interface  
+ `Material UI`: the CSS framework.  
+ `React Js`: Front end framework  
+ `Redux`: a predictable state container for JavaScript apps.
+    
+## UI Design
+The mockups for the app design can be viewed [here](#)
+
+## REST API Docs
+The Api documentation is done using swagger. click [here](#) to View Barefoot Nomad API Documentation
+
+## Tests setup and execution
+The project use MOCHA and CHAI for testing.  
+All tests are located in `./src/tests/**.test.js`  
+To run tests run `npm test`
+
+## Installation and usage instructions
+
+Clone the repository using: git clone https://github.com/andela/the_spinners-backend.git  
```

Comment:
Add instructions on cloning and renaming `.env-sample` file to `.env` and setting the right values

Reply:
Okay, Done.